lms/static/sass/course/base/_base.scss

```diff
@@ -59,7 +59,6 @@ table {
 
 a {
   &:hover, &:focus {
-    color: $pink;
```

Comment:
@andy-armstrong Any objections to removing this old magenta color from August, 2012? It's causing issues on the wiki, and I'm pretty sure we have tried to override this color everyplace.

Unfortunately I can't remove the "text-decoration: none !important;" part because it does seem to be in use (due to the !important).